includes/Context.php

```diff
@@ -187,6 +206,42 @@ public function is_beta() {
 		return false !== strpos( GOOGLESITEKIT_VERSION, 'beta' );
 	}
 
+	/**
+	 * Checks whether AMP content is being served.
+	 *
+	 * @since 1.0.0
+	 *
+	 * @return bool True if an AMP request, false otherwise.
+	 */
+	public function is_amp() {
+		return function_exists( 'is_amp_endpoint' ) && is_amp_endpoint();
+	}
+
+	/**
+	 * Get the current AMP mode.
```

Comment:
@aaemnnosttv See my other comment, this should be third-person. I'll update it quickly, because the rest is looking good.